lib/datasets/seaborn-data.rb

```diff
@@ -1,6 +1,15 @@
 module Datasets
   class SeabornData < Dataset
     URL_FORMAT = "https://raw.githubusercontent.com/mwaskom/seaborn-data/master/%{name}.csv".freeze
+    
+    # Show available dataset names.
+    def self.list
+      # This list is acquired by:
+      # $ ruby -rpycall -e "p PyCall.import_module('seaborn').get_dataset_names"
+      [ 'anagrams', 'anscombe', 'attention', 'brain_networks', 'car_crashes', 'diamonds',
+        'dots', 'exercise', 'flights', 'fmri', 'gammas', 'geyser', 'iris', 'mpg', 'penguins',
+        'planets', 'taxis', 'tips', 'titanic']
+    end
```

Comment:
Could you split this feature to another pull request to merge the `nil` key support?

FYI: We should implement this feature as a dataset like `Datasets::RdatasetsList`: https://github.com/red-data-tools/red-datasets/blob/master/lib/datasets/rdatasets.rb#L5

Reply:
I should have done so. I will make it as Datasets::SeabornDataList .
Thank you for your comment.